.gitignore

```diff
@@ -1,22 +1,27 @@
-*.sw[pon]
-cmd/foo.go
-unsorted.*
+# Editor Leftovers
 .DS_Store
-data/*
-testdir
```

Comment:
can we keep testdir with a comment?

Reply:
Yes, good call -- it's gonna prevent unwanted commits for leftover data dirs

[57984e4](https://github.com/gnolang/gno/pull/1944/commits/57984e4202ac452e6f1058194c1ec0daff793166)